es-ds-components/components/es-rating.vue

```diff
@@ -85,32 +53,42 @@ const update = (value: number) => {
             </span>
         </div>
     </div>
-    <rating
-        v-else
-        :model-value="roundedRating"
-        :cancel="false"
-        :readonly="readOnly"
-        v-bind="$attrs"
-        :pt="{
-            root: {
-                class: 'bg-transparent rounded-0 text-orange rating reactive',
-            },
-            item: {
-                class: 'reactiveStar',
-            },
-        }"
-        @update:model-value="update">
-        <template #officon>
-            <icon-star-empty
-                :width="width"
-                :height="height" />
-        </template>
-        <template #onicon>
-            <icon-star-full
-                :width="width"
-                :height="height" />
-        </template>
-    </rating>
+    <div v-else>
+        <rating
+            v-model="model"
+            :cancel="false"
+            :readonly="readOnly"
+            v-bind="$attrs"
+            :pt="{
+                root: {
+                    class: 'bg-transparent rounded-0 text-orange rating reactive',
+                },
+                item: (options) => {
+                    return {
+                        class: [
+                            {
+                                reactiveStar: true,
+                                focused: options.context.focused,
+                                'fade-focus': options.context.active && options.context.focused,
```

Comment:
Is this being used?

Reply:
It was a start to getting fade to work, I'll revert it.